File: 88-merge-sorted-array/88-merge-sorted-array.py
class Solution:
    def merge(self, nums1: List[int], m: int, nums2: List[int], n: int) -> None:
        """
        Do not return anything, modify nums1 in-place instead.
        """
        x=nums1.copy()
        nums1.clear()
        for i in range(m):
            nums1.append(x[i])
        for i in range(n):
            nums1.append(nums2[i])
        nums1.sort()
            
            
        
            
        
        

File: 88-merge-sorted-array/88-merge-sorted-array.cpp
class Solution {
public:
    void merge(vector<int>& nums1, int m, vector<int>& nums2, int n) {
        int i;
        int l=nums1.size();
        for(i=0;i<n;i++)
        {
           nums1[l-i-1]=nums2[i];
        }
        sort(nums1.begin(),nums1.end());
    }
};